File: src/main/java/ru/dragonestia/apocalypse/item/CopperClusterItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class CopperClusterItem extends Item implements AdvItem {

    public CopperClusterItem(Integer meta, int count) {
        super(ApocalypseID.COPPER_CLUSTER, meta, count, "Copper cluster");
    }

    @Override
    public String getRuName() {
        return "Медная руда";
    }

    @Override
    public String getIcon() {
        return "textures/items/cluster/copper";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/CopperClusterItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class CopperClusterItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.COPPER_CLUSTER, 0, count, "Copper cluster"), AdvItem {

    override fun getRuName(): String = "Медная руда"

    override fun getIcon(): String = "textures/items/cluster/copper"

}